gui/system/models.py

```diff
@@ -329,6 +308,27 @@ class Advanced(Model):
         help_text=_("Global password to unlock SED disks."),
         verbose_name=_("SED Password"),
     )
+    adv_sysloglevel = models.CharField(
+        max_length=120,
+        choices=choices.SYS_LOG_LEVEL,
+        default="f_info",
+        verbose_name=_("Syslog level"),
+        help_text=_("Specifies which messages will be logged by "
+                    "server. INFO and VERBOSE log transactions that "
+                    "server performs on behalf of the client. "
+                    "f_is_debug specify higher levels of debugging output. "
+                    "The default is f_info."),
+    )
+    adv_syslogserver = models.CharField(
+        default='',
+        blank=True,
+        max_length=120,
+        verbose_name=_("Syslog server"),
+        help_text=_("Specifies the server and port syslog messages "
```

Comment:
This is a duplicate of 0050_move_syslog.py and should be reworded the same way.

Reply:
I've removed `help_text` from the migration.